kouncil-frontend/libs/common-login/src/lib/login/common-login.component.ts

```diff
@@ -1,44 +1,54 @@
-import {ChangeDetectionStrategy, Component, EventEmitter, Output} from '@angular/core';
+import {ChangeDetectionStrategy, Component, EventEmitter, Input, Output} from '@angular/core';
 import {FormBuilder, FormControl, FormGroup, Validators} from "@angular/forms";
 import {User} from "./user";
+import {Backend} from "@app/common-model";
 
 @Component({
   selector: 'app-common-login',
   template: `
-    <div class="icon-login-container">
+    <div class="icon-login-container"
+         [ngClass]="backend === 'SERVER' ? 'icon-login-container-desktop' : 'icon-login-container-demo'">
       <mat-icon aria-hidden="false" class="icon-login">person</mat-icon>
     </div>
     <div class="main-login">
       <form (ngSubmit)="login()" class="login-form" [formGroup]="form">
         <span class="login-info">Log in to your account</span>
 
+        <span *ngIf="firstTimeLogin" class="first-time-login">
+          Default user credentials:
+          <br>
+          username: admin
+          <br>
+          password: admin
+        </span>
+
         <app-common-login-field [fieldName]="'username'"
                                 [control]="getControl('username')"
                                 [fieldType]="'text'"
                                 [autocomplete]="'username'"
                                 [label]="'Login'"
                                 [icon]="'person'"></app-common-login-field>
-        <br>
 
         <app-common-login-field [fieldName]="'password'"
                                 [control]="getControl('password')"
                                 [fieldType]="'password'"
                                 [autocomplete]="'current-password'"
                                 [label]="'Password'"
                                 [icon]="'lock'"></app-common-login-field>
-        <br>
         <button mat-button disableRipple class="action-button-white" type="submit">
           Login
         </button>
       </form>
     </div>
   `,
   changeDetection: ChangeDetectionStrategy.OnPush,
-  styleUrls: ['./common-login.component.scss']
+  styleUrls: ['./common-login.component.scss', '../common-login.scss']
```

Comment:
i do not like it. I like when 1 component have only 1 css file. You can always sahre css with mixins/import css inside component css file.